docker/Dockerfile.tests

```
@@ -58,28 +57,80 @@ RUN apt-get update && \
         libxml2 \
         libxml2-dev \
         locales \
+        lsof \
         make \
+        maven \
+        openjdk-8-jdk \
+        openjdk-8-jre \
         openssh-client \
         perl \
         postgresql \
         postgresql-10-pgtap \
         postgresql-contrib \
         postgresql-server-dev-10 \
-        python-minimal \
+        python2.7-dev \
+        python2.7-minimal \
+        rabbitmq-server \
         redis-server \
         runit \
         runit-systemd \
         sudo \
         unzip \
+        virtualenv \
         yarn && \
     rm -rf /var/lib/apt/lists/* && \
     rm google-chrome-stable_current_amd64.deb && \
-    rm nodejs_10.17.0-1nodesource1_amd64.deb
+    rm nodejs_10.17.0-1nodesource1_amd64.deb && \
+    update-java-alternatives -s java-1.8.0-openjdk-amd64 && \
+    systemctl disable rabbitmq-server
+
+ENV PG_AMQP_TAG v0.4.1
+
+RUN git clone --branch $PG_AMQP_TAG --depth 1 https://github.com/omniti-labs/pg_amqp.git && \
+    cd pg_amqp && \
+    make && \
+    make install && \
+    cd /home/musicbrainz
+
+ENV SOLR_VERSION 7.7.2
+ENV SOLR_HOME /opt/solr/server/solr
+
+RUN curl -sLO https://www.apache.org/dist/lucene/solr/$SOLR_VERSION/solr-$SOLR_VERSION.tgz && \
+    tar xzf solr-$SOLR_VERSION.tgz solr-$SOLR_VERSION/bin/install_solr_service.sh --strip-components=2 && \
+    ./install_solr_service.sh solr-$SOLR_VERSION.tgz && \
+    systemctl disable solr
 
 RUN wget -q -O - https://cpanmin.us | perl - App::cpanminus && \
     cpanm Carton JSON::XS
 
-RUN curl -sLO https://chromedriver.storage.googleapis.com/78.0.3904.70/chromedriver_linux64.zip && \
+ENV MB_SOLR_TAG v3.1.1
+
+# Steps taken from https://github.com/metabrainz/mb-solr/blob/master/Dockerfile
+RUN sudo -E -H -u musicbrainz git clone --recursive https://github.com/metabrainz/mb-solr.git && \
+    cd mb-solr && \
+    git checkout $MB_SOLR_TAG && \
+    cd mmd-schema/brainz-mmd2-jaxb && \
+    mvn install && \
+    cd ../../mb-solr && \
+    mvn package -DskipTests && \
+    mkdir -p /opt/solr/lib $SOLR_HOME && \
+    cp target/mb-solr-0.0.1-SNAPSHOT-jar-with-dependencies.jar /opt/solr/lib/ && \
+    cd .. && \
+    cp -R mbsssss $SOLR_HOME/mycores/ && \
+    sed -i'' 's|</solr>|<str name="sharedLib">/opt/solr/lib</str></solr>|' $SOLR_HOME/solr.xml && \
+    mkdir $SOLR_HOME/data && \
+    chown -R solr:solr /opt/solr/ && \
+    cd /home/musicbrainz
+
+ENV SIR_COMMIT 7c4c50b
+
+RUN sudo -E -H -u musicbrainz git clone https://github.com/metabrainz/sir.git && \
+    cd sir && \
+    git reset --hard $SIR_COMMIT && \
```

Doesn’t it require sudo too?

```suggestion
    sudo -E -H -u git reset --hard $SIR_COMMIT && \
```

---

I guess not since it works without it, but that would be more correct & avoid any permission issues in the future.